uint8 steering_motor_id
int16 steering_motor_angle